### ISSUE TITLE
Could not remove remote packages

### ISSUE THREAD

**Issue by uilianries** [conan-io/conan#3174]
To help us debug your issue please explain:
Hi!

I could not remove any package from Bintray using Conan remove command, even for version 1.5.1. Some hours ago status.bintray.com reported about an error, but is stable now. So I don't have sure what could be happened.

Conan version: 1.5.2

Steps to reproduce:
```
$ conan add remote bincrafters https://api.bintray.com/conan/bincrafters/public-conan
$ conan user -r bincrafters -p ${CONAN_PASSWORD} ${CONAN_LOGIN_USERNAME} 
$ conan remove -r bincrafters --outdated zstd/1.3.4@bincrafters/stable
```

- [X] I've read the [CONTRIBUTING guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
- [X] I've specified the Conan version, operating system version and any tool that can be relevant.
- [X] I've explained the steps to reproduce the error or the motivation/use case of the question/suggestion.

**Comment by lasote** [comment]
What kind of error is it raising?

**Comment by danimtb** [comment]
Just tested with a remote in bintray and worked well:

`conan remove libwebp/1.0.0@danimtb/testing -r upload_repo --outdated`

https://bintray.com/danimtb/public-conan/libwebp%3Adanimtb

**Comment by uilianries** [comment]
Sorry, I forgot to paste the log. I just created this issue before to sleep :joy: 

Anyway, I'm running the same command now and it's working. So the problem was related to Bintray.

The original error was HTTP 400 (bad request). I used CONAN_LOGGING_LEVEL=10 to check what happened, the only answer when I tried to remove was ERROR 400.

As 400 is related to a bad request I though that could be a problem with client version x server version.

Thanks again.
